### ISSUE TITLE
dataloader config

### ISSUE THREAD

**Issue by XieJiajian2024** [juhha/3D-mADUNet#2]
Thank you for your open source work, which inspired me a lot. I am trying to use the method for my own data set (3D data of dx512x512png). I hope to write a dataloader to specify the training validation and test set as the model input, and delete the data opt and postprocessing config of yaml file, but I noticed that there are source key and target key in the train opt. I don't know whether writing a dataloader myself will affect the training process.
Can you give me some suggestions? Thank you again for your great work！

**Comment by juhha** [comment]
Hi! Thanks for showing interest in this.

I'd say the easiest way is probably pre-process and save slides of 2D images into nifti file format.
After the conversion, you won't have to worry about data I/O.

Another way is creating your own data I/O pipeline, which may be time-consuming and slower in I/O pipeline.

Thanks!

**Comment by XieJiajian2024** [comment]
Thank you for your reply!  I'll try later
